Desktop/components/JASP/Controls/TextField.qml

```diff
@@ -128,8 +128,7 @@ TextInputBase
 			height:				parent.height + jaspTheme.jaspControlHighlightWidth
 			width:				parent.width  + jaspTheme.jaspControlHighlightWidth
 			color:				"transparent"
-			border.width:		control.acceptableInput ? 0 : 3 //See comment below
-			border.color:		jaspTheme.red // Needed when the QML file has wrong default value
+			border.width:		0
```

Comment:
Might be useful to mention here that it is set programmatically because that is not quite obvious from qml...